online.athn.browser.gnome.json

```diff
@@ -0,0 +1,51 @@
+{
+    "app-id": "online.athn.browser.gnome",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "44",
+    "sdk": "org.gnome.Sdk",
+    "sdk-extensions": [
+        "org.freedesktop.Sdk.Extension.rust-stable"
+    ],
+    "command": "reference-browser-gnome",
+    "finish-args": [
+        "--share=ipc",
+        "--socket=wayland",
+        "--socket=fallback-x11",
+        "--device=dri",
+        "--share=network"
+    ],
+    "build-options": {
+        "prepend-path": "/usr/lib/sdk/rust-stable/bin",
+        "build-args": [
+            "--share=network"
+        ],
+        "env": {
+            "RUST_BACKTRACE": "1"
+        },
+        "strip": true
+    },
+    "modules": [
+        {
+            "name": "reference-browser-gnome",
+            "buildsystem": "simple",
+            "build-commands": [
+                "cargo build --release",
+                "install -D target/release/reference-browser-gnome /app/bin/reference-browser-gnome",
+                "mkdir -p /app/share/glib-2.0/schemas /app/share/applications /app/share/metainfo /app/share/icons/hicolor/scalable/apps",
+                "cp src/resources/online.athn.browser.gnome.gschema.xml /app/share/glib-2.0/schemas/",
+                "glib-compile-schemas /app/share/glib-2.0/schemas/",
+                "cp metadata/online.athn.browser.gnome.desktop /app/share/applications/online.athn.browser.gnome.desktop",
+                "cp metadata/online.athn.browser.gnome.metainfo.xml /app/share/metainfo/online.athn.browser.gnome.metainfo.xml",
+                "cp metadata/online.athn.browser.gnome.svg /app/share/icons/hicolor/scalable/apps/online.athn.browser.gnome.svg"
```

Comment:
lines37-39: if you use the `install` command, then you don't need line 34. (also applied to line 35)

Reply:
you still should use install.